sky/engine/tonic/dart_args.h

```diff
@@ -17,7 +17,10 @@ namespace blink {
 class DartArgIterator {
  public:
   explicit DartArgIterator(Dart_NativeArguments args)
```

Comment:
Can't we just add a default value for start_index instead of having a separate overload?